keyboards/melgeek/z70ultra/z70ultra.c

```diff
@@ -0,0 +1,152 @@
+#include "z70ultra.h"
+#include "is31fl3741.h"
```

Comment:
This shouldn't be needed. 
```suggestion
```

Reply:
Cause I need use this function 'IS31FL3741_set_pwm_buffer' to set indicator LEDs.